src/components/app.js

```diff
@@ -1,9 +1,24 @@
 import React from 'react';
 import '../assets/css/app.scss';
+import {Route} from 'react-router-dom';
+import About from './about';
+import Contact from './contact';
+import Footer from './footer';
+import Header from './header';
+import Home from './home';
+import Services from './services';
 
 const App = () => (
-    <div>
-        <h1>Sweet Corner</h1>
+    <div className = "app">
+        <div className = "container">
+            <Header/>
+                <Route path = "/about" component = {About} />
+                <Route path = "/contact" component = {Contact} />
+                <Route exact path = "/" component = {Home} />
+                <Route path = "/services" component = {Services} />
+            <Footer/>
```

Comment:
Notice that the Footer and Header components have no associated Route component. This is so that these components will be present on all possible paths. For these components, we want them present on all pages, so there is no need to utilize React Router in conjunction with them. Great job!